IdlePlus/src/Patches/Guilds/ClanActionWebhooks.cs

```diff
@@ -0,0 +1,25 @@
+namespace IdlePlus.Utilities {
+	/// <summary>
+	/// Constants for clan action webhook strings used throughout the application.
+	/// </summary>
+	public static class ClanActionWebhooks {
+		public const string SkillingTicketReceived = "skilling-ticket-received";
+		public const string ItemSentToVault = "item-sent-to-vault";
+		public const string ItemsSentToVault = "items-sent-to-vault";
```

Comment:
I'm not sure we need two separate webhooks here, since the result is effectively the same. Wouldn't it make more sense to just use `items-sent-to-vault` for both single and multiple items?

Reply:
Sure. I just wasn't sure the best way to serialize multiple items. Will combine them and make a comma separated list.

Reply:
After looking again, I think it makes sense to have them be separate since the shape of the payload varies between them. In the singular action, we get the `item_id` and `quantity` separately, where we get a get a `Dictionary<item_id, quantity>`. Because the webhook payload value is expected to be a string, I serialized dictionary of multiple items to JSON so the receiver could parse it.

I could combine them and always provide a serialized dictionary in the body, but that didn't feel great.

Happy to go either direction, or some other I haven't thought of.

Reply:
I like the idea of just sending the items as a serialized dictionary. That way, the site implementing the webhooks only need to handle a single endpoint for depositing items, rather than maintaining two separate ones.